src/test/java/com/yahoo/sketches/theta/TestPerformanceTheta.java

```diff
@@ -0,0 +1,245 @@
+
+  public static void main(String[] args) throws Exception {
+
+    TestPerformanceTheta test = new TestPerformanceTheta();
+
+    if(args.length<6) {
+      test.LOG.info("Missing arguemnts: java com.yahoo.sketches.concurrent.theta"
+          + ".TestPerformanceTheta <concurrencyType> <family> <#writers> <#readers> <seconds> "
+          + "<print>\n e.g., java "
+          + "com.yahoo.sketches.theta.TestPerformanceTheta CONCURRENT QUICKSELECT 4 4 "
+          + "30 true");
+      System.exit(0);
+    }
+
+    int i=0;
+    String concurrencyType = args[i++];
+    String family = args[i++];
+    int writers = Integer.parseInt(args[i++]);
+    int readers = Integer.parseInt(args[i++]);
+    int time = Integer.parseInt(args[i++]);
+    boolean print = Boolean.parseBoolean(args[i++]);
+
+    if (print) {
+      test.LOG.info("writers = " + writers);
+    }
+
+    test.setUp(concurrencyType, family);
+    test.runTest(CONCURRENCY_TYPE.valueOf(concurrencyType), writers, readers, time);
+
+    test.LOG.info("Done!");
+
+    System.exit(0);
+  }
+
+  public void setUp(String concurrencyType, String family) throws Exception {
+
+    WritableMemory mem;
+    UpdateSketch sketch;
+    UpdateSketch sketchToInit=null;
+    UpdateSketchBuilder usb = Sketches.updateSketchBuilder().setNominalEntries(K);
+    if(family.equalsIgnoreCase(OFF_HEAP)){
+      mem = WritableMemory.wrap(new byte[(K * 16) + 24]);
+      sketch = usb.build(mem);
+    } else {
+      sketch = usb.setFamily(Family.valueOf(family)).build();
+    }
+
+    switch (CONCURRENCY_TYPE.valueOf(concurrencyType)) {
+
+    case CONCURRENT:
+      if(family.equalsIgnoreCase(OFF_HEAP)){
+        gadget_ = ConcurrentThetaFactory.createConcurrentUpdateSketch(sketch);
+      } else {
+        gadget_ = ConcurrentThetaFactory.createConcurrentUpdateSketch(Family.valueOf(family));
+      }
+      sketchToInit = ConcurrentThetaFactory.createConcurrentThetaContext(
+          (ConcurrentUpdateSketch) gadget_);
+      LOG.info("=============================================CONCURRENT_THETA"
+          + "===========================================");
+      break;
+    case LOCK_BASED:
+      gadget_ = new LockBasedUpdateSketch(sketch);
+      sketchToInit = gadget_;
+      LOG.info("=============================================LOCK_BASED_THETA"
+          + "===========================================");
+      break;
+    case BASELINE:
+      gadget_ = sketch;
+      sketchToInit = sketch;
+      LOG.info("=============================================BASELINE_THETA"
+          + "===========================================");
+      break;
+    default:
+      String msg = concurrencyType + "is not a valid concurrency type, please choose "
+          + "CONCURRENT/LOCK_BASED/BASELINE";
+      LOG.info(msg);
+      throw new RuntimeException(msg);
+    }
+
+    for (long i = 0; i < 10000000; i++) {
+      sketchToInit.update(i);
+      if((i%100000)==0){
+        System.out.print(".");
+      }
```

Comment:
Other that drawing a line of dots, what is the purpose of this? 
Why not print to LOG rather than to System.out ?